spec/features/consumer/shopping/checkout_spec.rb

```diff
@@ -119,12 +119,17 @@
         end
       end
 
-      it "doesn't tell about previous orders" do
+      it "shows only applicable content" do
+        # Unifying several specs here to speed up test run.
+
+        # it doesn't tell about previous orders
         expect(page).to have_no_content("You have an order for this order cycle already.")
-      end
 
-      it "doesn't show link to terms and conditions" do
+        # doesn't show link to terms and conditions
```

Comment:
IMO it doesn't improve the readability of the RSpec matcher on the line below, so I'd remove it.